app/code/Magento/Backend/Block/Dashboard/Graph.php

```diff
@@ -344,7 +344,7 @@ public function getChartUrl($directUrl = true)
                 }
             } else {
                 // EXTENDED ENCODING
-                for ($j = 0; $j < sizeof($thisdataarray); $j++) {
+                for ($j = 0, $jMax = sizeof($thisdataarray); $j < $jMax; $j++) {
```

Comment:
Could you also replace sizeof with count?
It's alias http://php.net/manual/en/function.sizeof.php